web/src/components/tasks/LicenseTask.test.tsx

```diff
@@ -451,11 +448,48 @@ describe("<LicenseTask />", () => {
       it("displays NOT_FOUND error alert when license status cannot be found", async () => {
         useMockBusiness({
           licenseData: generateLicenseData(
-            {},
+            { lastUpdatedISO: "" },
+            {
```

Comment:
dust - based on you b/e logic, we'd return an empty object , or licenses that are not tied to the current task, therefore ,shouldn't this be an empty object?